env-defaults

```diff
@@ -338,9 +341,24 @@ PROMETHEUS_REQUESTS_EPHEMERAL_STORAGE=300Mi
 PROMETHEUS_LIMITS_CPU=128m
 PROMETHEUS_LIMITS_MEMORY=300Mi
 PROMETHEUS_LIMITS_EPHEMERAL_STORAGE=500Mi
+PROMETHEUS_ALERT_MANAGER_REQUESTS_CPU=500m
+PROMETHEUS_ALERT_MANAGER_REQUESTS_MEMORY=500M
+PROMETHEUS_ALERT_MANAGER_LIMITS_CPU=1
+PROMETHEUS_ALERT_MANAGER_LIMITS_MEMORY=1Gi
+
+
 PROMETHEUS_PROMETHEUS_SCRAPE_INTERVAL=15s
 PROMETHEUS_POD_MANAGER_SCRAPE_INTERVAL=15s
 
+# Set this to the ARN of the prometheus-alerts-local AWS SNS topic from harmony sandbox
+# (in your .env file) if you plan to deploy Prometheus for local development.
+# Your email address must also be subscribed to the topic in order to receive alerts.
+PROMETHEUS_ALERTS_ARN=
```

Comment:
This one is tricky - I guess we want to allow people to deploy Prometheus to their docker-desktop cluster, but then use sandbox resources for alerting?

If the goal is to make that possible I guess we should also add a PROMETHEUS_ALERTS_ROLE_ARN and use that role ARN when deploying Prometheus.

Reply:
My hope was that people could see actual emails when developing locally.